src/tutorial/fem.md

```diff
@@ -55,12 +55,12 @@ $$
 
 where $u_h$ is the finite element approximation with degrees of freedom (unknown
 coefficients) $c_j$, and $\varphi_j$ are known _basis functions_. The FEM basis
-are typically piecewise-polynomial functions on a given computational mesh,
+functions are typically piecewise-polynomial functions on a given computational mesh,
 which are only non-zero on small portions of the mesh.
 
 <img src="../img/fem1.png" width="300">
 
-With finite elements, the mesh can be totally unstructured, curved and
+With finite elements, the mesh can be totally unstructured, curved, and
 non-conforming:
 
 <img src="../img/fem2.png" width="300">
```

Comment:
fem2.png is broken for me locally, even though the image file is in the correct directory.

Reply:
Fixed